modules/openapi-generator/src/main/resources/typescript-nestjs-server/README.md

```diff
@@ -0,0 +1,56 @@
+#  OpenApi Generator _typescript-nestjs-server_
+
+Usage: The generated output is intended to be its own module, that can be imported into your NestJS App Module. You do not need to change generated files, just import the module and implement the API
+
+Example usage (with the openapi sample `petstore.yaml`):
+
+1. Invoke openapi-generator
+```
+openapi-generator-cli.jar generate -i petstore.yaml -g typescript-nestjs-server -o api-module/
+```
+2. implement the contracts from `api-module/api`
+
+`handlers/PetService.ts`:
+```typescript
+import { Pet, ApiResponse } from "models";
+import { Observable } from "rxjs";
+import { PetApi } from "../api";
+import { Inject, Injectable } from "@nestjs/common";
+
+@Injectable()
+export class PetService implements PetApi {
+addPet(pet: Pet, request: Request): Pet | Promise<Pet> | Observable<Pet> {
+    throw new Error("Method not implemented.");
```

Comment:
can you please fix the indentation of all typescript-code in this markdown file?